### ISSUE TITLE
Assess commuter data story suitability for dashboard

### ISSUE THREAD

**Issue by LiamOSullivan** [BuildingCityDashboards/bcd-dd-v2.1#544]
Does the planned commuter story for the model transfer to the dashboard?

**Comment by samstehle** [comment]
tl;dr: Yes, the commuter story will transfer.
The necessary components for implementation are:
   -spatial views (already a dashboard component)
   -static json files with specific data for the story
   -large json files arranged temporally to access with...
   -time slider to scan through daily/weekly patterns of data
   -ability to 'turn the page' to the next part of the story, adding new text and data map data

Commuter stories are fictional accounts of Dublin residents who interact with data on their trips to and from work. Their interactions may be actual - using a real-time app to access public transit locations - or suggested - the characters think about a phenomena they witness normally, and the data bears it out.
The stories and data facilitating them are contrived but based on normal patterns, then users are given the ability to interact with the underlying data in its true form, usually spatially and often with a time slider.
I have considered 'interactive graphic novel' as a suitable analogy for the story, with a short narrative and timestamp on story elements, with interactive components between story elements.
Dashboard medium also allows introduction of other visualisation types beyond the spatial. Charts and imagery are possible, but weren't considered for the tabletop model implementation

**Comment by LiamOSullivan** [comment]
Relate to #309

**Comment by samstehle** [comment]
a potential example of how commuter stories might be adapted to the dashboard.
This is more text-heavy than the table projection
https://storymaps.arcgis.com/stories/795fa0328cb2497a8f960b5f3e8d2c7a
